packages/services/hmi-server/build.gradle

```diff
@@ -29,13 +29,19 @@ dependencies {
 	implementation "io.quarkus:quarkus-smallrye-openapi"
 	implementation "io.quarkus:quarkus-arc"
 	implementation "io.quarkus:quarkus-logging-json"
+	// Multipart file upload/download
+	implementation 'org.glassfish.jersey.media:jersey-media-multipart:3.1.1'
+	implementation 'org.glassfish.jersey.core:jersey-server:3.1.1'
 
 	// Persistence
 	implementation "io.quarkus:quarkus-hibernate-orm"
 	implementation "io.quarkus:quarkus-jdbc-postgresql"
 	implementation "io.quarkus:quarkus-hibernate-orm-panache"
 	annotationProcessor "io.quarkus:quarkus-panache-common"
-
+	// Multipart file upload/download
+	implementation 'org.glassfish.jersey.media:jersey-media-multipart:3.1.1'
```

Comment:
phew we're going to have a massive amount of conflicts....

Unrelated to above, why is this imported twice?